packages/@sanity/default-layout/src/sideMenu/SideMenu.tsx

```diff
@@ -2,90 +2,136 @@
 
-      <div className={styles.inner}>
-        <div className={styles.header}>
-          <div className={styles.headerMain}>
-            <div className={styles.userProfile}>
-              <div className={styles.userAvatarContainer}>
-                <UserAvatar size="medium" userId="me" />
-              </div>
-              <div className={styles.userProfileText}>{user.name || user.email}</div>
-            </div>
+      <InnerCard display="flex" height="fill" $open={isOpen} shadow={1}>
+        <Card borderBottom>
+          <Stack space={3} padding={[3, 3, 4]}>
+            <Flex align="center">
+              <Flex flex={1} align="center" paddingRight={2}>
+                <Flex flex={1} align="center">
+                  <Box>
+                    <UserAvatar size="medium" userId="me" />
+                  </Box>
+                  <Box flex={1} marginLeft={2} title={user.name || user.email}>
+                    <Text textOverflow="ellipsis">{user.name || user.email}</Text>
+                  </Box>
+                </Flex>
+              </Flex>
 
-            <div className={styles.closeButtonContainer}>
-              <Button
-                icon={CloseIcon}
-                kind="simple"
-                onClick={onClose}
-                padding="small"
-                tabIndex={tabIndex}
-                title="Close menu"
-              />
-            </div>
-          </div>
+              <Box>
+                <Button
+                  ref={closeButtonRef}
+                  icon={CloseIcon}
+                  onClick={onClose}
+                  tabIndex={tabIndex}
+                  title="Close menu"
+                  mode="bleed"
+                />
+              </Box>
+            </Flex>
 
-          {HAS_SPACES && (
-            <div className={styles.datasetSelectContainer}>
-              <DatasetSelect isVisible={isOpen} />
-            </div>
-          )}
-        </div>
+            {HAS_SPACES && (
+              <Box>
+                <DatasetSelect tabIndex={tabIndex} />
+              </Box>
+            )}
+          </Stack>
+        </Card>
 
-        <div className={styles.toolSwitcher}>
+        <Box flex={1} overflow="auto" padding={[3, 3, 4]}>
           <ToolMenu
             activeToolName={activeToolName}
             direction="vertical"
             isVisible={isOpen}
             onSwitchTool={onSwitchTool}
-            router={router}
             tools={tools}
           />
-        </div>
+        </Box>
 
-        <div className={styles.menuBottom}>
-          <div className={styles.signOutButton}>
-            <Button icon={SignOutIcon} kind="simple" onClick={onSignOut} tabIndex={tabIndex}>
-              Sign out
-            </Button>
-          </div>
-        </div>
-      </div>
-    </Layer>
+        <Card paddingX={2} paddingY={3} borderTop>
+          <Stack>
+            <Button
+              justify="flex-start"
+              icon={LeaveIcon}
```

Comment:
This should be  `iconRight={LeaveIcon}`

![image](https://user-images.githubusercontent.com/406933/133989778-d6d527e0-0229-44a4-9af7-e9b7bef9d1ae.png)